ocs_ci/ocs/resources/pod.py

```diff
@@ -194,7 +197,7 @@ def exec_s3_cmd_on_pod(self, command, mcg_obj=None):
             else None,
         )
 
-    def copy_to_pod(self, src_path, target_path, container=None):
+    def copy_to_pod_rsync(self, src_path, target_path, container=None):
```

Comment:
I added a comment on this in the old PR and it looks like it was not addressed. adding it here again.

```You have renamed the existing function, please make sure that there is no regression with this change. For instance this function is used by javasdk_pod_fixture
```

Reply:
discussed, ++ done